cli/azd/internal/repository/infra_confirm.go

```diff
@@ -399,6 +401,17 @@ azureDepPrompt:
 				AuthUsingConnectionString: authType == scaffold.AuthType_PASSWORD,
 				AuthUsingManagedIdentity:  authType == scaffold.AuthType_TOKEN_CREDENTIAL,
 			}
+		case appdetect.AzureDepEventHubsForKafka:
+			authType, err := i.chooseAuthType(ctx, azureDepName)
+			if err != nil {
+				return err
+			}
+			spec.AzureEventHubsForKafka = &scaffold.AzureDepEventHubsForKafka{
+				Name:                      azureDepName,
```

Comment:
Delete `Name` because it's not used. Related PR: https://github.com/azure-javaee/azure-dev/pull/17